packages/edit-site/src/components/add-new-template/new-template.js

```diff
@@ -181,55 +187,68 @@ export default function NewTemplate( {
 						{ isCreatingTemplate && (
 							<TemplateActionsLoadingScreen />
 						) }
-						<NavigableMenu className="edit-site-new-template-dropdown__popover">
```

Comment:
`<NavigableMenu>` doesn't seem to be needed since `<DropdownMenu>` already renders it under the hood.